prometheus_remediation/job_runner.py

```diff
@@ -137,12 +145,16 @@ def run_job_from_alert(event: PrometheusKubernetesAlert, params: JobParams):
     job.create()
 
     if params.notify:
-        event.add_enrichment([MarkdownBlock(f"Created job from alert: *{job_name}*.")])
+        event.add_enrichment([MarkdownBlock(f"Created Job from alert: *{job_name}*.")])
 
     if params.wait_for_completion:
-        wait_until_job_complete(job, params.completion_timeout)
-        job = hikaru.from_dict(job.to_dict(), cls=RobustaJob)  # temporary workaround for https://github.com/haxsaw/hikaru/issues/15
-        pod = job.get_single_pod()
-        event.add_enrichment([
-            FileBlock("job-runner-logs.txt", pod.get_logs())
-            ])
+        try:
+            wait_until_job_complete(job, params.completion_timeout)
+            job = hikaru.from_dict(job.to_dict(), cls=RobustaJob)  # temporary workaround for https://github.com/haxsaw/hikaru/issues/15
+            pod = job.get_single_pod()
+            event.add_enrichment([
+                FileBlock("job-runner-logs.txt", pod.get_logs())
+                ])
+        except Exception as e:
```

Comment:
You might swallow other exceptions (not just the one you are looking for)
Verify it is the right exception and rethrow it if it isn't the exception you are looking for.

See an example of how to do this here - https://stackoverflow.com/a/13531310/495995